api/migrations/versions/e511f343cb08_complete_removal_of_continuous_journal.py

```diff
@@ -0,0 +1,27 @@
+"""Complete removal of continuous journal
+
+Revision ID: e511f343cb08
+Revises: 81526c9dbb6f
+Create Date: 2024-04-02 19:49:24.440649
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e511f343cb08'
+down_revision = '81526c9dbb6f'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.execute("""DELETE FROM pg_enum
+        WHERE enumlabel = 'CONTINUOUS_JOURNAL' 
+        AND enumtypid = (
+        SELECT oid FROM pg_type WHERE typname = 'event_type'
+        )""")
+
+
+def downgrade():
+    op.execute("COMMIT")
+    op.execute("ALTER TYPE event_type ADD VALUE 'CONTINUOUS_JOURNAL'")
```

Comment:
This enum wasn't removed from the db. Need to do this so it doesn't keep popping up when creating new migrations.